code/modules/paperwork/photography.dm

```diff
@@ -218,6 +220,7 @@
 		var/image/IM = getFlatIcon(the_turf.loc)
 		if(IM)
 			res.Blend(IM, blendMode2iconMode(the_turf.blend_mode),xoff,yoff)
```

Comment:
```suggestion
			CHECK_TICK
			res.Blend(IM, blendMode2iconMode(the_turf.blend_mode),xoff,yoff)
```
The get flat icon just before this did a bunch of blends to worth checking tick before this blend.

Also try testing this by merging with master to get the getflaticon fixes pr in and you can better evaluate if this resolves any issues with large pictures (you may need to opt in to the more expensive get flat icon only for testing - not something the cameras should generally opt into unless we find cases where the extra work is needed for photos)

Reply:
Also whitespace may be funky on mobile - not sure why this suggestion diff is red lining the existing code - just want a new line with check tick.